resources/js/Components/MismatchRow.vue

```diff
@@ -199,13 +194,21 @@ export default defineComponent({
 
 <style lang="scss">
 @import '~@wmde/wikit-tokens/dist/_variables.scss';
+@import '~@wikimedia/codex-design-tokens/theme-wikimedia-ui.scss';
+@import '../../sass/link.scss';
 
-    .wikit-Link.break-line-link {
-      width: 100%;
-    }
-    .wikit-Link__content {
+  .cdx-docs-link {
```

Comment:
This overwrites the `.break-line-link` class rules but not all wikit links have that class, I would advise to keep these rules separate from the mixin.